content/faq/how-to-clean-install-preserve

```diff
@@ -0,0 +1,57 @@
+---
+title: How do I do a clean install of LBRY while preserving important data?
+category: setup
+---
+
+Under certain circumstances, particularly if you've tested earlier beta versions, it may be recommended that you perform a clean installation of LBRY.  This generally involves uninstalling the current LBRY application, removing various directories/files and then reinstalling the latest version of LBRY. Different operating systems use different folder locations and [this post on directories](https://lbry.io/faq/lbry-directories) will help if you cannot locate the right files or folders.  
+
+After uninstalling LBRY, the working directories are left intact so that the application can be easily reinstalled without losing your application data and wallet file. Application data is in the lbrynet folder (LBRY folder for Mac) and it includes various database and configuration files.  Wallet and blockchain data is in the lbryum folder. Typically, you will want to save your wallet and a handful of configuration/database files when performing a clean install in order to keep your important data intact.
+
+Note: after a clean install, you may be prompted again for your email. This is normal. As always, you can email [help@lbry.io](mailto:help@lbry.io) or reach out to us on Slack if you encounter any trouble.
+
+## Windows
+1. Reboot your PC (this is to ensure that no LBRY processes are running)
+2. Uninstall the LBRY application by accessing "Add or Remove Programs" via the Control Panel
+3. Type `%appdata%` into an Explorer window to find the working directories
+4. Delete the entire LBRY folder
+5. If performing a clean install, delete the lbryum and lbrynet folders also **(!!THIS WILL DELETE YOUR WALLET AND DATA!!)** and skip to step 8 
+6. Navigate to the lbryum folder and delete the `blockchain_headers` file
+7. Navigate to the lbrynet folder and delete all items while leaving the following:
+ 1. `blobfiles folder` - stores encrypted downloaded files which are used for hosting
+ 2. `blobs.db` - reference data for the blob files which are used for hosting purposes
+ 3. `lbryfile_info.db` - Downloads and Publishes data
+ 4. `blockchainname.db` - Supports downloads data
```

Comment:
Got it. I would assume a new install id would be generated and they would be re-prompted for their email/ verification code - is that what we want? If the code is now expired, they will need one re-sent.